tests/testthat/test_grade_learnr.R

```diff
@@ -44,7 +44,20 @@ test_grade_learnr <- function(user_code,
                last_value = last_value)
 }
 
-test_that("grade learnr", {
+test_that("Grade learnr check_result", {
+  expect_correct(
+    test_grade_learnr(
+      user_code = "4",
+      check_code = "check_result(
+          pass_if(~ .result == 4, 'yes. you did it')
```

Comment:
@schloerke this is why I changed that `graded` value from `message` to `final_result$message` I was getting a random praise otherwise.

However, i need to change it back to `message` becuase the message does not show up when I actually test it out in a learnr tutorial

Reply:
Can test with something like

```r
check_result(correct = "{message}", pass_if(~TRUE))
```